makefiles/deprecated_modules.inc.mk

```diff
@@ -2,5 +2,15 @@
 # Keep this list ALPHABETICALLY SORTED!!!!111elven
 DEPRECATED_MODULES += event_thread_lowest
 DEPRECATED_MODULES += gnrc_netdev_default
+DEPRECATED_MODULES += gnrc_pktbuf_cmd # use shell_cmd_gnrc_pktbuf instead
+DEPRECATED_MODULES += gnrc_udp_cmd # use shell_cmd_grnc_udp instead
+DEPRECATED_MODULES += heap_cmd # use shell_cmd_heap instead
+DEPRECATED_MODULES += i2c_scan # use shell_cmd_i2c_scan instead
+DEPRECATED_MODULES += md5sum # use shell_cmd_md5sum instead
+DEPRECATED_MODULES += nice # use shell_cmd_nice instead
+DEPRECATED_MODULES += random_cmd # use shell_cmd_random instead
 DEPRECATED_MODULES += sema_deprecated
+DEPRECATED_MODULES += shell_commands # use shell_cmds_default instead
```

Comment:
What's the reasoning behind this rename?

Reply:
IMO having both `shell_cmds` and `shell_commands` is confusing, as it is not obvious by the name what their relationship is. I have to admid that the relationship of `shell_cmds` and `shell_cmds_default` is not super clear by their names only. But at least after knowing what they do it should be easy to not confuse the two.